ntc_templates/templates/huawei_vrp_display_isis_peer.textfsm

```diff
@@ -1,4 +1,4 @@
-Value SYSTEM_ID ((\d+.\d+.\d+)|\S+)
+Value List SYSTEM_ID (\S+)
```

Comment:
This changes the data structure and would have to wait for a major version bump.